src/PrestaShopBundle/Resources/config/routing/admin/improve/design/mail_theme.yml

```
@@ -81,3 +81,11 @@ admin_mail_theme_send_test_module_mail:
     _controller: PrestaShopBundle:Admin\Improve\Design\MailTheme:sendTestMail
     _legacy_link: AdminMailTheme:rawModuleLayout
     _legacy_controller: AdminMailTheme
+
+admin_mail_theme_translate_body:
+  path: /translateBody
```

Never use camelcase for urls
```suggestion
  path: /translate-body
```

---

Alright, what about the other ones in came case? (in this file) I should switch them as well, but isn't it a BC?

---

You can change them and add alias for old urls.

---

But there is no alias feature in Symfony routing, this implies creating new routes but they still will be accessible unless we redirect them via another controller

---

You can create 
```yml
admin_mail_theme_translate_body_alias:
  path: /translateBody
```
And add a comment. Will be removed in the next release / major

---

sadly I don't see any alternative Too bad this was not seen in 1760
I'll add the aliases in a separate file to make this a bit cleaner